x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -109,10 +109,12 @@ export const TransactionDistribution: FunctionComponent<Props> = (
     bucketIndex,
   } = props;
 
+  /* eslint-disable-next-line react-hooks/exhaustive-deps */
   const formatYShort = useCallback(getFormatYShort(transactionType), [
     transactionType,
   ]);
```

Comment:
What's the violation here?

Reply:
![image](https://user-images.githubusercontent.com/35559/83888599-144f0c80-a718-11ea-8c56-7a47550ee78e.png)
My guess is that its trying to analyze the source code of the first param, but since its not a function expression, it can't.
The following is valid:
```
  const formatYShort = useCallback(() => getFormatYShort(transactionType), [
    transactionType,
  ]);
```